src/current/_includes/v23.2/ui/statement-fingerprints.md

```diff
@@ -0,0 +1,44 @@
+{% if page.cloud == true %}
+  {% capture link_prefix %}../{{site.current_cloud_version}}/{% endcapture %}
+  {% assign page_prefix = "" %}
+{% else %}
+  {% assign link_prefix = "" %}
+  {% assign page_prefix = "ui-" %}
+{% endif %}
+
+### Example
+
+See [View historical statement statistics and the sampled logical plan per fingerprint]({{ link_prefix }}crdb-internal.html#view-historical-statement-statistics-and-the-sampled-logical-plan-per-fingerprint).
+
+## SQL statement fingerprints
+
+The Statements page displays SQL statement fingerprints.
+
+A _statement fingerprint_ represents one or more SQL statements by replacing literal values (e.g., numbers and strings) with underscores (`_`). This can help you quickly identify frequently executed SQL statements and their latencies.
+
+For multiple SQL statements to be represented by a fingerprint, they must be identical aside from their literal values and placeholders.
+
+These SQL statements:
+
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (380, 11, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (192, 891, 20)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (784, 452, 78)`
+
+have the fingerprint `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES (_, _, _)`
+
+These SQL statements:
+
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES ($1, $2, 11098)`
+- `INSERT INTO new_order(product_id, customer_id, transaction_id) VALUES ($3, $4, 300)`
```

Comment:
Not sure if we want to include an invalid SQL statement as an example - placeholders should start at 1, but I agree this illustrates the fingerprinting mechanism.